vulyk/blueprints/gamification/models/events.py

```diff
@@ -53,7 +52,8 @@ class EventModel(Document):
             'user',
             {
                 'fields': ['answer'],
-                'unique': True
+                'unique': True,
```

Comment:
As I stated in PM the change causes slightly different behaviour thanks to the magic within `mongoengine`.
Previous state has two indices:
1. `[_cls, answer] unique`
2. `[answer] unique`

Current model provides only a single one:
1. `[answer] unique sparse`

If you by any reason rely on `_cls` key, you might as well tune new index up.